Dockerfile

```diff
@@ -1,22 +1,28 @@
-FROM python:3
+FROM python:3.9 as base
 
-RUN mkdir /bandersnatch
-RUN mkdir /conf && chmod 777 /conf
-ADD setup.cfg /bandersnatch
-ADD setup.py /bandersnatch
-ADD requirements.txt /bandersnatch
-ADD requirements_swift.txt /bandersnatch
-ADD README.md /bandersnatch
-ADD CHANGES.md /bandersnatch
-COPY src /bandersnatch/src
+FROM base as builder
+RUN mkdir /install
+WORKDIR /install
+ADD requirements_swift.txt /install
+ADD requirements.txt /install
+RUN pip install --target="/install" --upgrade pip setuptools wheel
+RUN pip install --target="/install" \
+    -r requirements.txt \
+    -r requirements_swift.txt
+
+
+FROM python:3.9-slim
 
-# OPTIONAL: Include a config file
-# Remember to bind mount the "directory" in bandersnatch.conf
-# Reccomended to bind mount /conf - `runner.py` defaults to look for /conf/bandersnatch.conf
-# ADD bandersnatch.conf /etc
+COPY --from=builder /install /usr/local/lib/python3.9/site-packages
```

Comment:
Is there some dynamic way so we can no hard code to 3.9?

Reply:
don't think so, main reason version is pinned up there
I'm happy to be explicit, not too much work to tune to 3.8 or 3.10  :)

Reply:
Yeah, it will just lag, we'll forget / miss it when next version comes out etc. I try to avoid that.

If we can't I'll open an issue to look into it and merge for now as the size wins are worth the work.